management/virtualmachine/client.go

```diff
@@ -118,6 +119,25 @@ func (vm VirtualMachineClient) GetRole(cloudServiceName, deploymentName, roleNam
 	return role, nil
 }
 
+// AddRole adds a Virtual Machine to a deployment of Virtual Machines, where role name = VM name
+// See https://msdn.microsoft.com/en-us/library/azure/jj157186.aspx
+func (vm VirtualMachineClient) AddRole(cloudServiceName string, deploymentName string, role Role) (management.OperationID, error) {
+	if cloudServiceName == "" {
```

Comment:
The Travis CI build is failing due to an unrelated error... "The command "go get -u github.com/golang/lint/golint" failed and exited with 1 during ." Any idea how can I kick off the build again? Thanks!